pkg/kube/inject/webhook_test.go

```diff
@@ -992,9 +992,13 @@ func createWebhook(t testing.TB, cfg *Config) (*Webhook, func()) {
 	env := model.Environment{
 		Watcher: mesh.NewFixedWatcher(&m),
 	}
+	watcher, err := NewFileWatcher(configFile, valuesFile)
+	if err != nil {
+		cleanup()
```

Comment:
nit - switch to t.Cleanup(cleanup) one time instead of on every return

Reply:
thank you, good to know about this